src/client/eppo-client.ts

```diff
@@ -185,6 +185,10 @@ export default class EppoClient implements IEppoClient {
     this.configurationStore = configurationStore;
   }
 
+  public setIsObfuscated(isObfuscated: boolean) {
+    this.isObfuscated = isObfuscated;
+  }
+
```

Comment:
by the way @greghuels this is the bad pattern I was trying to remember where this method `setIsObfuscated` is on the `EppoClient` now but if we implement lets say `JSEppoClient` from the interface it won't have it available, urgh!

Reply:
I'm not following. Why would we need a `JSEppoClient` if we already have `EppoClient`?

Reply:
oh, you mean if we also forget to add the signature to the interface, then it won't be available? Yeah, that's annoying.